OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -25,13 +25,16 @@
 #		$ bash build.sh
 #
 
-FROM oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
 
 # Labels
 LABEL "provider"="Oracle"                                               \
       "issues"="https://github.com/oracle/docker-images/issues"         \
       "volume.data"="/opt/oracle"
 
+# Install yum package manager
+RUN microdnf -y install yum
+
 # Install packages to install and run Management Agent
 RUN set -eux; \
```

Comment:
What is the purpose of `set -eux`? Also, why not just use `microdnf` to install these packages? Why are you installing `yum`?